app/src/ui/history/commit-list-item.tsx

```diff
@@ -153,12 +153,16 @@ export class CommitListItem extends React.PureComponent<
           onMouseUp={this.onMouseUp}
         >
           <div className="info">
-            <RichText
-              className="summary"
-              emoji={this.props.emoji}
-              text={commit.summary}
-              renderUrlsAsLinks={false}
-            />
+            {commit.summary.length > 0 ? (
```

Comment:
Nit..just coding preference so feel free to ignore.. IMHO tertiary operations it the html are harder to read. I would tend towards keeping the RichText and passing in a variable of `const commitSummary = commit.summary.length > 0 ? commit.summary : 'Empty commit message'` and similarly pass in a classname variable to the RichText that conditionally has the `empty-summary`. Only annoying thing with this approach is doing the empty check twice.. but I think I still prefer it. :/